### ISSUE TITLE
Local Instance

### ISSUE THREAD

**Issue by In3x0rabl3** [orcasecurity-research/AIGoat#5]
Would it be difficult to convert this project into a local project without utilizing AWS?

**Comment by mikejl** [comment]
Agree.  A VM would be great to be able to use with our college NETLABs.

**Comment by shirs-orca** [comment]
It is in our backlog at the moment :)
